storage-access.bs

```diff
@@ -293,6 +308,49 @@ ISSUE: Write this section.
 
 ISSUE: Write this section.
 
+<h2 id="automation">Automation</h2>
+
+For the purposes of user-agent automation and application testing, this document defines the following [=extension command=] for the [[WebDriver]] specification.
+
+<h3 id="set-storage-access-command">Set Storage Access</h3>
+
+<table>
+    <tbody>
+    <tr>
+        <th>HTTP Method</th>
+        <th>URI Template</th>
+    </tr>
+    <tr>
+        <td>POST</td>
+        <td>/session/{session id}/storageaccess</td>
+    </tr>
+    </tbody>
+</table>
+
+The <dfn export>Set Storage Access</dfn> [=extension command=] modifies the storage access policy for the [=current browsing context=].
+
+The [=remote end steps=] are:
+
+
+1. Let |blocked| be the result of [=getting a property=] from |parameters| named `blocked`.
+1. If |blocked| is not a [=boolean=] return a [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+1. Let |origin| be the result of [=getting a property=] from |parameters| named `origin`.
+1. If |origin| is not the [=serialization of an origin=] return a [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+1. If |origin| is an [=opaque origin=] return a [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+1. Let |embedded site| be the result of [=obtain a site|obtaining a site=] from |origin|.
```

Comment:
Adding the ability to specify this per origin seems useful. Would it be useful to allow a mechanism to specify all origins (e.g. equiv to "blockallthirdpartycookies" preferences) as well?